test/decompile/basic.txt

```diff
@@ -127,32 +121,23 @@ export function f(a:int, b:int):int {
   if (e < 10.0) { 1[4]:int = 2[3]:int + 5 }
   f(a + g_b, 9);
   loop L_b {
-    block B_c {
-      if (if (0) { 1 } else { 2 }) break B_c;
-      continue L_b;
-    }
+    if (if (0) { 1 } else { 2 }) goto B_c;
+    continue L_b;
+    label B_c:
     if (1) continue L_b;
   }
   select_if(1, 2, 1 == 1);
-  block B_e {
-    block B_f {
-      block B_g {
-        block B_h {
-          0;
-          block B_i {
-            0;
-            br_table[B_f, B_g, B_h, B_i, ..B_e](a);
-            return 99;
-            unreachable;
-          }
-          return 100;
-        }
-        return 101;
-      }
-      return 102;
-    }
-    return 103;
-  }
+  br_table[B_f, B_g, B_h, B_i, ..B_e](a);
+  unreachable;
+  label B_i:
```

Comment:
Oh that's nicer.

Nit/Opinion: dedent the labels? Having them in the same column as the instructions makes it harder to see the control flow structure at a glance.

Reply:
Yeah, that wasn't trivial to do, so left it like this for now. That was also partially the motivation to add the `label` keyword. May revisit that later to dedent.